src/client/helpers/wObjInfluenceHelper.js

```diff
@@ -22,12 +22,12 @@ export const setInitialObjPercents = (linkedObjects, percentage) => {
   const averagePercent = Math.floor(100 / len);
   const firstPercent = 100 - averagePercent * (len - 1);
   const objPercentage = {
-    [first.id]: {
+    [first.id || first.author_permlink]: {
       percent: firstPercent,
     },
   };
   rest.forEach(obj => {
-    objPercentage[obj.id] = { percent: averagePercent };
+    objPercentage[obj.id || first.author_permlink] = { percent: averagePercent };
```

Comment:
You could use only 'id' if apply 'getClientWObj' to all linked objects

Reply:
++